NEWS.md

```diff
@@ -2,7 +2,7 @@
 
 ## MsCoreUtils 1.11.2
 
-- Nothing yet.
+- Reimplement `between` in C (see issue #105).
```

Comment:
Maybe also mention that you use symbol lookup for C calls for all functions calling C functions? Just that we have that also in the record.

Reply:
Done